Installation.md

```diff
@@ -85,6 +85,7 @@ To install Swift for TensorFlow, download one of the packages below and follow t
 
 | Download | Date |
 |----------|------|
+| [Windows](https://storage.googleapis.com/azure-pipelines-storage/Swift%20for%20TensorFlow/Windows/s4tf-windows-x64-27604-20200306.1.exe) | March 6, 2020 |
```

Comment:
Please re-order.

Reply:
They are currently in date order, do we want to change that?

Reply:
Yes, the date doesn't matter.

Reply:
Okay, shuffled the dates around to not make it look chronological, and reordered.